src/runtime/virtcontainers/qemu_arm64.go

```diff
@@ -100,6 +101,32 @@ func (q *qemuArm64) appendImage(ctx context.Context, devices []govmmQemu.Device,
 	return q.appendBlockImage(ctx, devices, path)
 }
 
+func (q *qemuArm64) appendNvdimmImage(devices []govmmQemu.Device, path string) ([]govmmQemu.Device, error) {
```

Comment:
Could you add a comment above to explain why we need it as a temporary workaround? So that we can revisit and remove it later on when qemu-aarch64 supports the new option.

Reply:
Thanks @bergwolf -,  done.